Exiled.API/Features/ExiledSerializable.cs

```diff
@@ -0,0 +1,47 @@
+// -----------------------------------------------------------------------
+// <copyright file="ExiledSerializable.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    using System;
+
+    /// <summary>
+    /// An attribute to easily manage Exiled features' behavior.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
+    public sealed class ExiledSerializable : Attribute
```

Comment:
Change it to ExiledSerializableAttribute based on the c# coding standart:

- Better readability
- The attribute will be the same, just the file name and class changes
- The file is inside the Features namespace which contains a lot of different features, indicating that this is the attribute for exiledserializable will help people when they are searching for something.

Reply:
It's longer, it's useless, it's not aesthetic, it sucks, no one uses C# coding standar**d**.
If you want better readability buy a new pair of glasses.

Reply:
👀

Reply:
To be fair, I agree with Nao, and we wouldn't exactly be the only project that doesn't follow that standard (looking at you [HarmonyPatch()])

Reply:
**HarmonyPatchAttribute** 🤣

Reply:
Yeah, but thats just the class name, when you use it you dont have to add attribute at the end.
Also, harmony uses a custom attribute derived from attribute and that class follows the standart.

https://github.com/pardeike/Harmony/blob/79d62b42c71d005b3cb1d94e741bfd4ce7e03a6b/Harmony/Public/Attributes.cs#L101

@NaoUnderscore @joker-119

Reply:
also, you can check below that it is used below without having to write attribute at the end.

Reply:
> **HarmonyPatchAttribute** 🤣

If you create a custom attribute you wont use it with the work attribute at the end.

TestAttribute : Attribute
you would use it as 
[Test]
Something

Reply:
Its just readiability, file name and class name, not usage

Reply:
Readability improves, nothing else changes. Everyone now suddenly knows that this class should be used as an attribute without looking at the inheritance.